index.html

```diff
@@ -1,14 +1,62 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="pages/index.css">
     <title>Pink App</title>
   </head>
   <body class="page">
-    <div class="page__container">
-      <p>The repository is created with the purpose of learning adaptive web design and team collaboration using Git and GitHub.</p>
+    <div class="page__container">    
+        <header class="header">
+                <div class="header__container">
+                  <a href="#" class="logo">
+                    <span class="visually-hidden">Логотип приложения Pink</span>
+                    <svg class="logo__icon logo__icon_screen_mobile" xmlns="http://www.w3.org/2000/svg" fill="currentColor"><path d="M.005 1.549c0-.336.13-.66.365-.9C.604.41.922.27 1.257.262H8.39c5.057 0 8.385 2.703 8.385 7.13v.06c0 4.846-4.033 7.358-8.802 7.358H2.538v6.801a1.28 1.28 0 0 1-.754 1.28 1.269 1.269 0 0 1-1.717-.758 1.28 1.28 0 0 1-.062-.522V1.549Zm8.067 10.968c3.706 0 6.14-1.994 6.14-4.986v-.06c0-3.21-2.394-4.875-5.961-4.875H2.588v9.881l5.484.04ZM21.593 1.389a1.282 1.282 0 0 1 .755-1.28 1.268 1.268 0 0 1 1.717.758c.056.168.077.346.062.522v20.222a1.28 1.28 0 0 1-.754 1.28 1.27 1.27 0 0 1-1.717-.758 1.282 1.282 0 0 1-.063-.522V1.389ZM30.674 1.419A1.278 1.278 0 0 1 31.926.173h.348a1.535 1.535 0 0 1 1.281.737l13.6 17.4V1.36A1.239 1.239 0 0 1 48.389.122c.327 0 .64.13.871.362.231.232.36.546.36.874v20.292a1.14 1.14 0 0 1-.703 1.092 1.13 1.13 0 0 1-.449.085h-.129a1.695 1.695 0 0 1-1.311-.778L33.117 4.24v17.41a1.258 1.258 0 0 1-1.191 1.247 1.238 1.238 0 0 1-1.212-1.247V1.42h-.04ZM55.938 1.389a1.282 1.282 0 0 1 .755-1.28 1.268 1.268 0 0 1 1.717.758c.056.168.077.346.062.522v12.295L71.228.55c.262-.271.618-.432.994-.448a1.27 1.27 0 0 1 1.222 1.256 1.29 1.29 0 0 1-.418.898l-7.947 7.897 8.573 10.53c.224.26.348.593.348.937a1.312 1.312 0 0 1-.838 1.191 1.228 1.228 0 0 1-1.497-.493L63.3 11.859l-4.828 4.846v4.886a1.27 1.27 0 0 1-.78 1.17 1.259 1.259 0 0 1-1.648-.685 1.27 1.27 0 0 1-.096-.485V1.389h-.01Z"/></svg>
```

Comment:
@NazarPauk Would you please add `aria-hidden="true"` and `focusable="false"` to all SVG you got to your blocks? By doing that we indicate that it is decorative and should be ignored by screen readers.

Reply:
Done✅

Reply:
@NazarPauk Awesome. Let's not forget a SVG inside your burger button 😉

Reply:
> Done✅

Done✅

Reply:
> @NazarPauk Awesome. Let's not forget a SVG inside your burger button 😉

Done✅